### PR TITLE
Player Movement & Input System Completed

### PR DESCRIPTION
**Pull request by marytheproducer**

### CODE REVIEW

File: EEG Game/Assets/Scripts/Input/InputManager.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    private static InputManager instance;
+
+    public static InputManager Instance
+    {
+        get { return instance; }
+    }
+
+    private void Awake()
+    {
+        // Make InputMager a Singleton
```
Comment (MJE10) [inline EEG Game/Assets/Scripts/Input/InputManager.cs]: "InputMager" typo?

**Review by MJE10 — CHANGES_REQUESTED** [review]
See possible typo in InputManager.cs
- Mouse sensitivity seems extremely high, hard to control where I look
- There are still some assets with the DragonSlayer naming in Assets/Content, is the plan to rename or remove these at some point or no?

File: EEG Game/Assets/Scripts/Input/InputManager.cs (hunk 1 above)
Comment (marytheproducer) [reply EEG Game/Assets/Scripts/Input/InputManager.cs]: Oh my bad let me fix that also yee I'll look into mouse sensitivity real quick

Comment (marytheproducer) [reply EEG Game/Assets/Scripts/Input/InputManager.cs]: I think we can keep the dragonslayer assets for testing

**Review by MJE10 — APPROVED** [review]
That feels much better

**Comment by MJE10** [comment]
PR moved to #3

**Pull request closed without merging**